modules/gltf/gltf_document.cpp

```diff
@@ -857,6 +857,7 @@ Error GLTFDocument::_parse_buffer_views(Ref<GLTFState> p_state) {
 	if (!p_state->json.has("bufferViews")) {
 		return OK;
 	}
+	const Vector<Vector<uint8_t>> &buffersData = p_state->buffers;
```

Comment:
Godot style is to use `snake_case` for variable names.

```suggestion
	const Vector<Vector<uint8_t>> &buffers_data = p_state->buffers;
```